package.json

```diff
@@ -47,6 +47,8 @@
     "lodash.throttle": "^4.1.1",
     "node-fetch": "^2.6.0",
     "subscriptions-transport-ws": "^0.9.16",
+    "util": "^0.12.4",
+    "vscode": "^1.1.37",
```

Comment:
I believe we don't need to install this. To be confirmed @jaroszpiotr91 @codergirl-al